main/src/cgeo/geocaching/maps/google/v2/GoogleMapActivity.java

```diff
@@ -252,7 +253,18 @@ protected void onActivityResult(final int requestCode, final int resultCode, fin
 
     @Override
     public void showFilterMenu(final View view) {
-        MapUtils.openFilterActivity(this, mapBase.getFilterContext(), mapBase.getCaches());
+        FilterUtils.openFilterActivity(this, mapBase.getFilterContext(), mapBase.getCaches());
     }
 
+    @Override
+    public boolean showFilterList(final View view) {
+        return FilterUtils.openFilterList(this, mapBase.getFilterContext());
+    }
+
+    @Override
+    public void refreshWithFilter(final GeocacheFilter filter) {
```

Comment:
refresh of the map is currently not working correct

Reply:
Those cases are very tricky to find, I usually find them only by finding a reproducable case, then set a breakpoint in "MapUtils.filter" (or now "FilterUtils.filter") and see whether it is reached or not. It is hard to tell from looking at the code alone...

If you run in a dead end here, then I would need to get the stuff running on my local machine to dive in deep.

Reply:
Looks good for me now, probably it was bad internet-connection